app/src/main/java/com/example/mycalculator/mvp/view/CalculatorView.java

```diff
@@ -18,4 +20,18 @@ public void drawNumber(String number) {
         binding.screenTextExample.setText(number);
     }
 
+    @Override
+    public void showErrorDivision() {
+        binding.screenTextExample.setText(getContext().getString(R.string.error_message_in_division));
```

Comment:
remember to add a null check for the getContext()

Reply:
Done